tests/e2e/tour/tour.spec.js

```diff
@@ -103,6 +108,77 @@ test.describe('When loading the page with dontShowTour preference set to true',
+
+    await login(context)
+
+    await page.route(/collections.json/, async (route) => {
+      await route.fulfill({
+        json: singleCollection.body,
+        headers: singleCollection.headers
+      })
+    })
+
+    await page.goto('/search')
+  })
+
+  test('should see the tour slide for non-logged in users', async ({ page }) => {
+    // Start the tour by clicking the "Start Tour" button
+    page.getByRole('button', { name: 'Start tour' }).click()
+
+    // Start Tour View: Welcome to Earthdata Search
+    await expect(page.locator('.search-tour__welcome')).toContainText('Welcome to Earthdata Search!')
+    await page.click('button:has-text("Take the tour")')
+    await expect(page.locator('.search-tour__content').first()).toContainText('This area contains the filters used when searching for collections')
+
+    // Navigating to the slide for non-logged in users
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Search for collections')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Use the temporal filters to limit search results to a specific date')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Use the spatial filters')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Use Advanced Search parameters')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Choose a portal to refine search')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Refine your search further')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('A high-level description')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('To make more room')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Pan the map by ')
+
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('Use the map tools')
+
+    // Slide 12: Save Project Button
+    await page.keyboard.press('ArrowRight')
+    await expect(page.locator('.search-tour__content').first()).toContainText('to save a project using your current search criteria')
+
+    // Step 13: User Menu Dropdown
+    await page.keyboard.press('ArrowRight')
+    await page.waitForTimeout(500)
+    await expect(page.locator('.search-tour__content').first()).toContainText('Use this menu to set preferences, view saved projects')
+  })
+})
+
 test.describe('Joyride Tour Navigation', () => {
```

Comment:
I think making this a companion to your new describe block tells a better story of what we are testing. So the new one being "When logged in", this one should be "When not logged in".